Python Part/network_models.py

```diff
@@ -177,16 +178,12 @@ def model_seven(self):
         images_test = self.cnn.pool(images_test)
         images_test = flatten_all_images(images_test)
 
-        dnn = dnn_layers(input_neurons=images_train.shape[1], hidden_neurons=0, output_neurons=10)
-
-        for epoch in range(50):
-
-            back_accuracy = dnn.backward_without_hidden(images_train, self.labels_train)
-            forward_accuracy = dnn.forward_without_hidden(images_test,self.labels_test)
-
-            print(f"backward {back_accuracy}")
-            print(f"forward {forward_accuracy}")
-
+        fnn = fnn_layers(input_neurons=784, output_neurons=10)
+        fnn.evaluate_model(
+            epoch_size=self.epoch, alpha=self.alpha, images_test=self.images_test,
```

Comment:
pass `images_test` and `images_train` , instead of passing the original images `self.images_test` , `self.images_train`

Reply:
Okey, thanks for your review, I will get on it as soon as I can.